CHANGELOG.md

```diff
@@ -16,6 +17,119 @@
 - [v0.5.0](#v050---2020-08-06)
 <!-- END TOC -->
 
+-------------------------------------------------
+## v0.19.0 - 2022-01-25
+
+# Release notes
+
+* This release introduces a new feature called "index per namespace".
+    Enabling it makes fluentd log to indices in elasticsearch based on the namespace from which the container log originates.
+* Check out the [upgrade guide](https://github.com/elastisys/compliantkubernetes-apps/blob/main/migration/v0.18.x-v0.19.x/upgrade-apps.md) for a complete set of instructions needed to upgrade.
```

Comment:
Maybe just swap these two around to make it like the others, so the "Checkout the upgrade guide..." comes first.